examples/grpc-server-with-prometheus/prometheus/prometheus.yaml

```diff
@@ -25,6 +25,10 @@ scrape_configs:
   #     - targets: ['localhost:9090']
 
   - job_name: 'grpcserver'
-    scrape_interval: 2s
+    scrape_interval: 1s
```

Comment:
Worth to put a comment that the scrape interval is reduced just for example purposes? (:

Reply:
Yep. Because I run a loop which sends msg to the server every 3 seconds. I want to see the change in the monitor graph every 3 seconds. So I reduce the interval of scraping in prometheus

Reply:
That makes sense - just since this is example it is worth to not reproduce this on production without thinking.